request-manager/graph/grapher.go

```diff
@@ -31,13 +34,16 @@ func NewGrapher(specs spec.Specs, idGenFactory id.GeneratorFactory) *Grapher {
 // DoChecks performs graph checks for all sequences.
 func (gr *Grapher) DoChecks() (seqGraphs map[string]*Graph, seqErrors map[string]error) {
```

Comment:
Re func name: maybe `CheckSequences` or something. "Do" falls into the same realm as not using "Get" (https://golang.org/doc/effective_go.html#Getters). Also, imho, all functions "do" so it doesn't add anything in terms of clarity, etc.

Reply:
Code comment can say a little more. The import thing about this func is that it returns both the final/official "list" (it's really a map) of sequence graphs _and_ all graph check errors. And also important: are the two returns mutually exclusive? Normally, with `func() (<var>, err)` the two are mutually exclusive: if err != nil then caller must ignore <var>.